bats/fb-content-katello.bats

```diff
@@ -198,8 +199,3 @@ EOF
   skip # because of above
   puppet agent --test && grep -q Lorem /tmp/dummy
 }
-
-@test "Delete an Organization" {
```

Comment:
I moved this to another file to facilitate the upgrade testing. If we delete the organization all of the data gets deleted. In an upgrade, having pre-seeded data (in this case by using the content-tests) can illuminate more issues than an empty database due to our data manipulation that occurs during upgrade.